docs/UserGuide.md

```diff
@@ -150,6 +157,11 @@ The Calendar section provides a weekly view of all tuition sessions in TutorBudd
 The left and right arrow allows you to view past and future tuition sessions.
 Click the 'Today' button to be brought back to this week's view.
 
+<div markdown="block" class="alert alert-primary">
```

Comment:
I was thinking if this would be better if we changed it to the "View" section. Since "View" technically represents the entire tab functionality while putting it in `student` might not be as intuitive. What do you think?

![image](https://user-images.githubusercontent.com/24240408/114256322-201bf700-99eb-11eb-867a-6d2b9d41f876.png)

Reply:
My initial rationale was for the user to see all tabs before letting them know about this functionality.
But I also think it's better to put it up here in the view as you mentioned since it's more intuitive.